recipe/meta.yaml

```diff
@@ -16,12 +16,14 @@ source:
 
 build:
   number: {{ number }}
+  # for cuda, building with 9.2 is enough to be compatible with all later versions,
+  # since arrow is only using libcuda, and not libcudart.
   skip: true  # [cuda_compiler_version not in (undefined, "None", "9.2")]
```

Comment:
These comments are independent from the rest of the PR, but I thought I'd through them in nevertheless.

I had gotten this aspect wrong [myself](https://github.com/conda-forge/arrow-cpp-feedstock/pull/137#discussion_r425777174) at first, and was pointed to the discussions in #123 & #125. This is my best attempt at destilling the information into a short comment that might help another future reader of this recipe to figure out what's happening.